lib/ansible/modules/identity/cyberark/cyberark_credential.py

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/python
+# Copyright: (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+module: cyberark_credential
+short_description: Module for retrieval of CyberArk vaulted credential using PAS Web Services SDK through the Central Credential Provider
+author:
+ - Edward Nunez (@enunez-cyberark)
+ - Cyberark Bizdev (@cyberark-bizdev)
+ - erasmix (@erasmix)
+ - James Stutes (@jimmyjamcabd)
+version_added: 2.8
```

Comment:
We haven't enforced this via CI yet, though we are moving to str, rather than floats for `version_added`.
```suggestion
version_added: "2.8"
```

Reply:
commited